nexong/api/Event/tests.py

```diff
@@ -776,3 +776,121 @@ def test_update_lesson_event_by_volunteer(self):
         )
         self.assertEqual(response.status_code, status.HTTP_200_OK)
         self.lessonevent2.refresh_from_db()
+
+
+class PartnerEventApiViewSetTestCase(TestCase):
+    def setUp(self):
+        self.factory = APIRequestFactory()
+        self.partner3 = Partner.objects.create(
+            description="testdeprueba22", address="333 Elm St", birthdate="1996-05-05"
+        )
+        self.family2 = Family.objects.create(name="Familia Pedriza")
+        self.education_center2 = EducationCenter.objects.create(
+            name="San Francisco Pelano"
+        )
+        self.educator2 = Educator.objects.create(
+            description="testdeprueba4", birthdate="2000-04-22"
+        )
+        self.educator3 = Educator.objects.create(
+            description="testdeprueba5", birthdate="2001-06-21"
+        )
+        self.user2 = User.objects.create(
+            username="usuariotest",
+            email="usuariotets@gmail.com",
+            role=PARTNER,
+            partner=self.partner3,
+        )
+
+        self.token = Token.objects.create(user=self.user2)
+
+        self.student = Student.objects.create(
+            name="José Manuela",
+            surname="Colino",
+            education_center=self.education_center2,
+            is_morning_student=False,
+            activities_during_exit="",
+            status="PENDIENTE",
+            current_education_year="TRES AÑOS",
+            education_center_tutor="Don Pedro Picapiedra",
+            nationality="PeruANO",
+            birthdate="2017-04-21",
+            family=self.family2,
+        )
+        self.student2 = Student.objects.create(
+            name="Jesulin Antonio",
+            surname="De Ubrique",
+            education_center=self.education_center2,
+            is_morning_student=True,
+            activities_during_exit="",
+            status="PENDIENTE",
+            current_education_year="TRES AÑOS",
+            education_center_tutor="Don Pablo Motos",
+            nationality="España",
+            birthdate="2015-04-21",
+            family=self.family2,
+        )
```

Comment:
Using unrealistic data is one of the failure conditions for this subject... Please, change this.

Reply:
Done the changes!